### ISSUE TITLE
Java aplication crash when open a file chooser after play a video with vlcj in linux

### ISSUE THREAD

**Issue by WalkER14** [caprica/vlcj#607]
Im developing an aplication using java , javafx and vlcj to stream a video, so if a execute this line:  
new MediaPlayerFactory().newHeadlessMediaPlayer(), before open at least one time a file chooser the aplication crash showing this error:
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fbe52cd8d44, pid=3697, tid=140453784127232
#
# JRE version: Java(TM) SE Runtime Environment (8.0_77-b03) (build 1.8.0_77-b03)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.77-b03 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libpthread.so.0+0x9d44]  pthread_mutex_lock+0x4
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

But if i open a file chooser before running the line above the error never occur.

**Comment by caprica** [comment]
There is another issue posted here that sounds very similar, but I currently can not find it.

**Comment by caprica** [comment]
Maybe #477 and #353 relate.

**Comment by caprica** [comment]
Hopefully fixed by #664.
